roles/openshift_aws/defaults/main.yml

```diff
@@ -61,6 +61,46 @@ openshift_aws_vpc:
 #    - cidr: 172.31.16.0/20
 #      az: "us-east-1a"
 
+openshift_aws_create_dns: False
+openshift_aws_dns_provider: ''  # [ dyn, route53 ]
+openshift_aws_dns_zone: "{{ openshift_aws_clusterid }}.example.com"
+openshift_aws_dns_records:
+  # Pertains to inventory file key: openshift_master_cluster_public_hostname
+  'api':
```

Comment:
We can facilitate managing more than one zone if we embed the key zone, like so:
```yaml
openshift_aws_dns_records:
  'api'
  value: ...
  zone: "{{ openshift_aws_dns_zone }}"
...
```